test/post.js

```diff
@@ -12,8 +12,6 @@ try{
 }
```

Comment:
for some reason these authentication calls were put into the individual test pages. That's unnecessary with the included config file.